cypress/integration/plugins/dashboards-assistant/conversation_history_spec.js

```diff
@@ -0,0 +1,195 @@
+
+      // Open chat flyout
+      cy.get('body')
+        .then(($body) => $body.find('.llm-chat-flyout').length !== 0)
+        .then((chatFlyoutOpened) => {
+          if (!chatFlyoutOpened) {
+            cy.get('img[aria-label="toggle chat flyout icon"]').click();
+          }
+        });
+    });
+    after(() => {
+      // Close Chat bot
+      cy.get('body')
+        .then(($body) => $body.find('.llm-chat-flyout').length !== 0)
+        .then((chatFlyoutOpened) => {
+          if (chatFlyoutOpened) {
+            cy.get('img[aria-label="toggle chat flyout icon"]').click();
+          }
+        });
+    });
+    describe('panel operations', () => {
+      it('should toggle history list', () => {
+        cy.get('.llm-chat-flyout button[aria-label="history"]').click();
+        cy.get('.llm-chat-flyout-body')
+          .contains('Conversations')
+          .should('be.visible');
+
+        cy.get('.llm-chat-flyout button[aria-label="history"]').click();
+        cy.get('textarea[placeholder="Ask me anything..."]').should('exist');
+        cy.get('.llm-chat-flyout-body')
+          .contains('Conversations')
+          .should('not.be.visible');
+      });
+
+      it('should back to chat panel', () => {
+        cy.get('.llm-chat-flyout button[aria-label="history"]').click();
+        cy.get('.llm-chat-flyout')
+          .contains('Conversations')
+          .should('be.visible');
+
+        cy.get('.llm-chat-flyout-body').contains('Back').click();
+        cy.get('textarea[placeholder="Ask me anything..."]').should('exist');
+      });
+
+      it('should hide back button in fullscreen mode', () => {
+        cy.get('.llm-chat-flyout button[aria-label="fullScreen"]').click();
+        cy.get('.llm-chat-flyout button[aria-label="history"]').click();
+
+        cy.get('.llm-chat-flyout')
+          .contains('Conversations')
+          .should('be.visible');
+        cy.get('textarea[placeholder="Ask me anything..."]').should('exist');
+        cy.get('.llm-chat-flyout-body')
+          .contains('Back', { timeout: 3000 })
+          .should('not.exist');
+
+        // Back to default mode
+        cy.get('.llm-chat-flyout button[aria-label="fullScreen"]').click();
+        cy.get('.llm-chat-flyout button[aria-label="history"]').click();
+      });
+    });
+    describe('history item operations', () => {
+      const conversations = [];
+
+      before(() => {
+        // Create conversations data
+        cy.sendMessage({
+          input: {
+            type: 'input',
+            content: 'What are the indices in my cluster?',
+            contentType: 'text',
+          },
+        }).then((result) => {
+          if (result.status !== 200) {
+            throw result.body;
+          }
+          conversations.push(result.body);
+        });
+      });
+
+      after(() => {
+        // Clear created conversations in tests
+        conversations.map(({ conversationId }) =>
+          cy.deleteConversation(conversationId)
```

Comment:
is there a bulk delete API that we can use? are will there be no need to implement that API.